### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Drin Racaj, Nils Donatantonio and Esteban Giorgis

### DISCUSSION THREAD

**Discussion by EstebanGiorgis** [heig-vd-dai-course/heig-vd-dai-course#485] [Show and tell]
Le but de notre application est de réaliser un service chat en ligne entre plusieurs clients (avec donc l'intermédiaire d'un serveur). La communication d'un client vers le serveur se fera en TCP (unicast) et la communication en le serveur et les clients se fera en UDP.

**Comment by ludelafo** [comment]
Looks good to us!

**Comment by EstebanGiorgis** [comment]
Project done.

Link of our repository : https://github.com/EstebanGiorgis/DAI_Project_2

Commit hash : a5d5b62432e1fb69a7ad3fa4d6b4748ae5ba7130
